docs/docs/integrations/tools/hyperbrowser_browser_agent_tools.ipynb

```diff
@@ -0,0 +1,395 @@
+                "print(result)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### With Custom Session Options\n",
+                "\n",
+                "All tools support custom session options:\n"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [
+                {
+                    "name": "stdout",
+                    "output_type": "stream",
+                    "text": [
+                        "{'data': 'I have found that the react package was last published 11 hours ago. This is the most recently updated package I could find.', 'error': None}\n"
+                    ]
+                }
+            ],
+            "source": [
+                "result = tool.run(\n",
+                "    {\n",
+                "        \"task\": \"Go to npmjs.com, and tell me when react package was last updated.\",\n",
+                "        \"session_options\": {\n",
+                "            \"session_options\": {\"use_proxy\": True, \"accept_cookies\": True}\n",
+                "        },\n",
+                "    }\n",
+                ")\n",
+                "print(result)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### Async Usage\n",
+                "\n",
+                "All tools support async usage:\n"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [
+                {
+                    "name": "stdout",
+                    "output_type": "stream",
+                    "text": [
+                        "{'data': 'The page displays information about the \"Example Domain,\" stating that it is used for illustrative purposes and can be utilized without permission. There\\'s a link to \"More information...\" but no specific contact details are provided.', 'error': None}\n"
+                    ]
+                }
+            ],
+            "source": [
+                "async def browse_website():\n",
+                "    tool = HyperbrowserBrowserUseTool()\n",
+                "    result = await tool.arun(\n",
+                "        {\n",
+                "            \"task\": \"Go to npmjs.com, click the first visible package, and tell me when it was updated\"\n",
+                "        }\n",
+                "    )\n",
+                "    return result\n",
+                "\n",
+                "\n",
+                "result = await browse_website()"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Use within an agent\n",
```

Comment:
agent code should use langgraph prebuilt agent

see https://python.langchain.com/docs/integrations/tools/tavily_search/ for reference